crates/uv/tests/lock.rs

```diff
@@ -1719,13 +1720,14 @@ fn lock_conditional_dependency_extra() -> Result<()> {
 
     // Re-run with `--locked`.
     uv_snapshot!(context.filters(), context.lock().arg("--locked"), @r###"
-    success: true
-    exit_code: 0
+    success: false
+    exit_code: 2
     ----- stdout -----
 
     ----- stderr -----
     warning: `uv lock` is experimental and may change without warning
     Resolved 7 packages in [TIME]
+    error: The lockfile at `uv.lock` needs to be updated, but `--locked` was provided. To update the lockfile, run `uv lock`.
```

Comment:
These snapshots look wrong. I haven't investigated yet, but we probably shouldn't merge before fixing these.